app/src/testDemo/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppScreenSizesScreenshotTests.kt

```diff
@@ -139,29 +139,32 @@ class NiaAppScreenSizesScreenshotTests {
     }
 
     private fun testNiaAppScreenshotWithSize(width: Dp, height: Dp, screenshotName: String) {
+        // TODO: Remove this when https://issuetracker.google.com/issues/327149166 is fixed
```

Comment:
This issue is fixed in `1.7.0-alpha04`, so we should be able to remove this workaround now.

Reply:
Updated to `1.7.0-alpha05` and workaround removed. Just need to verify that the screenshots haven't changed.